roles/nova-common/tasks/main.yml

```diff
@@ -4,7 +4,7 @@
         createhome=yes generate_ssh_key=yes ssh_key_bits=4096
 
 - name: nova packages
-  apt: pkg=python-numpy
+  apt: pkg={{ nova.install_packages }}
```

Comment:
that's not quite going to work. 
```
  apt:
    pkg: "{{ item }}"
  with_items: nova.install_packages
```

Something like that should do the trick. You still have to form it as a with_items type loop, but just feed it in the single variable which is a list of items.